### PR TITLE
Feature/desgin-register

### PR DESCRIPTION
**Pull request by mahmmedn19**

### CODE REVIEW

File: app/src/main/res/layout/fragment_register.xml [hunk 1]
```diff
@@ -1,14 +1,143 @@
...
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <Button
+                android:id="@+id/button_sign_up"
+                style="@style/ButtonSignUpStyle"
+                android:text="@string/sign_up" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
```
Comment (AzizaHelmy) [inline app/src/main/res/layout/fragment_register.xml:118]: why this space!! 
you forget to use ctr+alt+l "DD

File: app/src/main/res/layout/fragment_register.xml [hunk 2]
```diff
@@ -1,14 +1,143 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/image_logo" />
+
+
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/card"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:text="@string/hello_blank_fragment" />
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
```
Comment (AzizaHelmy) [inline app/src/main/res/layout/fragment_register.xml:39]: use dimns resource file to get the sizes from it please
don't use any hard code!

File: app/src/main/res/layout/fragment_register.xml [hunk 3]
```diff
@@ -1,14 +1,143 @@
...
+
     <TextView
+        android:id="@+id/text_logo_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing_medium"
+        android:fontFamily="@font/notosans_medium"
+        android:text="@string/hello_honey"
+        android:textColor="@color/black_87"
+        android:textSize="@dimen/text_giant"
```
Comment (AzizaHelmy) [inline app/src/main/res/layout/fragment_register.xml:29]: i think we can make this as a sytle

File: app/src/main/res/layout/fragment_register.xml [hunk 4]
```diff
@@ -1,14 +1,143 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#FEC754"
```
Comment (AzizaHelmy) [inline app/src/main/res/layout/fragment_register.xml:7]: don't use any hard code
call the color from res we did

**Review by AzizaHelmy — CHANGES_REQUESTED** [review]
plz fix all this comments before merging,
keep going

File: app/src/main/res/layout/fragment_register.xml (hunk 4 above)
Comment (mahmmedn19) [reply app/src/main/res/layout/fragment_register.xml]: This not handle now because the screen needs the background custom shape  but I finish the screen

**Pull request merged**